### PR TITLE
Update requirements for python 3.11.3

### PR DESCRIPTION
**Pull request by kandeeban15**

### CODE REVIEW

**Comment by DamithaSenevirathne** [comment]
@kandeeban15 Shall we also remove the usages of previous versions in the docker file?

You need to update the following line 

https://github.com/miyurud/jasminegraph-prerequisites/blob/a6acffeeedd2113559d43e259657bfff26d8de8e/Dockerfile#L83

File: python_libs/requirements [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
+mock-5.1.0
+networkx-3.1
+numpy-1.24.3
+pbr-5.11.1
+protobuf-4.23.4
+scikit-learn-1.3.0
+scipy-1.11.1
+six-1.16.0
+tensorboard-2.13.0
+tensorflow-2.13.0
```
Comment (DamithaSenevirathne) [inline python_libs/requirements]: Shall we test the train command before updating the tensorflow version?

Comment (kandeeban15) [reply python_libs/requirements]: Done.

**Review by DamithaSenevirathne — APPROVED** [review]
LGTM

**Comment by Ishad-M-I-M** [comment]
@miyurud `Lint Code Base` failing as the default `failure-threshold` of `hadolint` ( The linter used in super-linter to lint Dockerfiles ) is set to `info`.
There are two options to fix this failure issue.
1.  Fix all the `info` and `warnings` shown.
2.  Set the `failure-threshold` of hadolint to `error`.
For this need to create a directory `linters` inside the `.github` folder and create a file named `.hadolint.yaml` with following content.
```
failure-threshold: error
```
This will override the default configurations of the hadolint and will pass the check
check: https://github.com/FYP-Auto-Scale-JasmineGraph/jasminegraph-prerequisites/pull/1

OR 

can discard the check failures by adding the lint ids to `igonre` section
```
ignored:
    - 
```
As they already doing in super-lint: https://github.com/super-linter/super-linter/blob/main/.github/linters/.hadolint.yaml

**Comment by miyurud** [comment]
> @miyurud `Lint Code Base` failing as the default `failure-threshold` of `hadolint` ( The linter used in super-linter to lint Dockerfiles ) is set to `info`. There are two options to fix this failure issue.
> 
> 1. Fix all the `info` and `warnings` shown.
> 2. Set the `failure-threshold` of hadolint to `error`.
>    For this need to create a directory `linters` inside the `.github` folder and create a file named `.hadolint.yaml` with following content.
> 
> ```
> failure-threshold: error
> ```
> 
> This will override the default configurations of the hadolint and will pass the check check: [FYP-Auto-Scale-JasmineGraph#1](https://github.com/FYP-Auto-Scale-JasmineGraph/jasminegraph-prerequisites/pull/1)
> 
> OR
> 
> can discard the check failures by adding the lint ids to `igonre` section
> 
> ```
> ignored:
>     - 
> ```
> 
> As they already doing in super-lint: https://github.com/super-linter/super-linter/blob/main/.github/linters/.hadolint.yaml

@kandeeban15 As we discussed offline lets follow the 2nd approach which @Ishad-M-I-M suggested. Because it will be difficult to treat the issues highlighted in the info logs. Furthermore, we cannot avoid having at least one info log.

**Pull request merged**